packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/template.ts

```diff
@@ -0,0 +1,11 @@
+import { html } from '@microsoft/fast-element';
+import type { RichTextMentionUsersView } from '.';
+
+export const template = html<RichTextMentionUsersView>`<span
```

Comment:
When doing display inline make sure there is no whitespace between elements in the template. Use the [anchor template](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/anchor/template.ts) as a reference and notice the pattern for opening and closing tags it uses to prevent any whitespace between tags

Reply:
Thanks, I have updated the template without whitespace between elements.